sources/packages/backend/apps/api/src/route-controllers/report/_tests_/e2e/report.institutions.controller.exportReport.e2e-spec.ts

```diff
@@ -255,4 +260,209 @@ describe("ReportInstitutionsController(e2e)-exportReport", () => {
+      {
+        student,
+        institution: collegeF,
+        institutionLocation: collegeFLocation,
+        programYear,
+      },
+      {
+        currentAssessmentInitialValues: {
+          workflowData: {
+            calculatedData: {
+              totalEligibleDependents: 2,
+              pdppdStatus: false,
+            },
+          } as WorkflowData,
+          assessmentDate: new Date(),
+          assessmentData: {
+            totalFederalAssessedResources: 10,
+            federalAssessmentNeed: 3,
+            totalProvincialAssessedResources: 5,
+            provincialAssessmentNeed: 11,
+            finalAwardTotal: 20,
+            finalFederalAwardNetCSGPAmount: 13,
+            finalFederalAwardNetCSGDAmount: 14,
+            finalProvincialAwardNetBCAGAmount: 15,
+            finalProvincialAwardNetSBSDAmount: 16,
+            finalProvincialAwardNetBCSLAmount: 17,
+            finalFederalAwardNetCSGFAmount: 18,
+            finalProvincialAwardNetBGPDAmount: 19,
+            totalAssessedCost: 50,
+          } as Assessment,
+        },
+        applicationData: {
+          indigenousStatus: "no",
+          citizenship: "canadianCitizen",
+          youthInCare: "no",
+          dependantstatus: "independant",
+        } as ApplicationData,
+      },
+    );
+    const payload = {
+      reportName: "Student_Unmet_Need_Report",
+      params: {
+        offeringIntensity: {
+          "Full Time": true,
+          "Part Time": true,
+        },
+        programYear: programYear.id,
+      },
+    };
+    const dryRunSubmissionMock = jest.fn().mockResolvedValue({
+      valid: true,
+      formName: FormNames.ExportFinancialReports,
+      data: { data: payload },
+    });
+    formService.dryRunSubmission = dryRunSubmissionMock;
+    const endpoint = "/institutions/report";
+    const institutionUserToken = await getInstitutionToken(
+      InstitutionTokenTypes.CollegeFUser,
+    );
+
+    // Act/Assert
+    await request(app.getHttpServer())
+      .post(endpoint)
+      .send(payload)
+      .auth(institutionUserToken, BEARER_AUTH_TYPE)
+      .expect(HttpStatus.CREATED)
+      .then((response) => {
+        const fileContent = response.request.res["text"];
+        const parsedResult = parse(fileContent, {
+          header: true,
+        });
+        expect(parsedResult.data).toEqual(
+          expect.arrayContaining([
+            {
+              "Application Disability Status": "no",
+              "Application Number": savedApplication.applicationNumber,
+              "Assessment Date": getISODateOnlyString(
+                savedApplication.currentAssessment.assessmentDate,
+              ),
+              "CIP Code":
+                savedApplication.currentAssessment.offering.educationProgram
+                  .cipCode,
+              "Citizenship Status":
+                savedApplication.currentAssessment.application.data.citizenship,
+              "Estimated BCAG": String(
```

Comment:
Not a blocker but what I believe to be more common in the source code is the use of the `.toString`.